redwood-layout-shared-test/src/commonMain/kotlin/app/cash/redwood/layout/AbstractFlexContainerTest.kt

```diff
@@ -223,6 +330,11 @@ abstract class AbstractFlexContainerTest<T : Any> {
     container.onEndChanges()
     verifySnapshot(container, "updated")
   }
+
+  /** We don't have assume() on kotlin.test. Tests that fail here should be skipped instead. */
+  private fun assumeTrue(b: Boolean) {
+    assertTrue(b)
+  }
```

Comment:
Delete? And delete call-sites? Presumably you did not create invalid variants when exploding them.

Reply:
I didn’t, but I want to keep the assume calls around for when we later do burst.